src/Filament/Resources/LinkCheckResource/Pages/ListLinkChecks.php

```diff
@@ -6,22 +6,63 @@
 use Filament\Resources\Pages\ListRecords;
 use Portable\FilaCms\Filament\Resources\LinkCheckResource;
 use Portable\FilaCms\Jobs\LinkChecker;
+use Filament\Notifications\Notification;
 
 class ListLinkChecks extends ListRecords
 {
     protected static string $resource = LinkCheckResource::class;
+    protected static ?string $title = 'Broken Links';
+    protected static string $view = 'fila-cms::admin.pages.broken-links';
+
 
     protected function getHeaderActions(): array
     {
         return [
             Actions\Action::make('scan')
                 ->label('Check Links')
-                ->action(fn () => $this->executeScan())
+                ->action(function () {
+                    $this->executeScan();
+                })
         ];
     }
 
     protected function executeScan()
     {
         LinkChecker::dispatch();
+
+        Notification::make()
```

Comment:
Are you aware that this will send at the start of the linkchecker running not after it has finished?